### PR TITLE
Update cd repo task

### PR DESCRIPTION
**Pull request by AsfaMumtaz**
Add new version of update cd repo task

### CODE REVIEW

**Comment by stakater-user** [comment]
@AsfaMumtaz Validation Successful

File: clustertask/github-update-cd-repo/templates/clustertask.yaml [hunk 1]
```diff
@@ -73,31 +73,45 @@ spec:
...
+          git clone
+          https://${GIT_USERNAME}:${GIT_PASSWORD}@github.com/$(GIT_ORGANIZATION)/$(CD_REPO_NAME)
           git config --global user.password ${GIT_PASSWORD}
           git config --global user.name ${GIT_USERNAME}
           git config --global user.email ${GIT_EMAIL}
           else
-          echo "$cd_repo_ssh_token" >> "~/.ssh/id_rsa"
-          git clone $params.CD_REPO_URL
+          git config --global user.name tekton-bot
+          git config --global user.email asfa@stakater.com
```
Comment (rasheedamir) [inline clustertask/github-update-cd-repo/templates/clustertask.yaml:97]: @AsfaMumtaz hardcoding your email?

**Review by rasheedamir — CHANGES_REQUESTED** [review]
plz check the comment @AsfaMumtaz

File: clustertask/github-update-cd-repo/templates/clustertask.yaml (hunk 1 above)
Comment (AsfaMumtaz) [reply clustertask/github-update-cd-repo/templates/clustertask.yaml:97]: Oops. Was just testing it out. This needs to have the stakater email.

**Review by rasheedamir — APPROVED** [review]

**Pull request merged**